### DISCUSSION TITLE
LEDs turning off, starting background effect, then returning to proper screen processing

### DISCUSSION THREAD

**Discussion by brfield94** [awawa-dev/HyperHDR#946] [General]
## Bug report, debug log and your config file (FULL LOGS ARE MANDATORY)
https://pastebin.com/0d5CBm35

#### Steps to reproduce
Intermittent issue with no direct cause. 

#### What is expected?
LEDs continue to match screen. 

#### What is actually happening?
LEDs turn to "Background Effect" (Waves with color)  when what I suspect is the grabber restarting. 

#### System

HyperHDR Server: 
- Build:           (HEAD detached at v20.0.0.0) (awawa-dev-9e960c9/52f12b4-1711236142)
- Build time:      Mar 23 2024 23:55:32
- Git Remote:      https://github.com/awawa-dev/HyperHDR
- Version:         20.0.0.0
- UI Lang:         auto (BrowserLang: en-US)
- Avail Capt:      Linux (V4L2)
- Database:        read/write

HyperHDR Server OS: 
- Distribution:   Debian GNU/Linux 12 (bookworm)
- Architecture:   arm64
- CPU Type:       Raspberry Pi 4 Model B Rev 1.5
- CPU Hardware:   c03115
- Kernel:         linux (6.6.31+rpt-rpi-v8 (WS: 64))
- Qt Version:     5.15.8
- Browser:        Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/18.0 Safari/605.1.15 

- Grabber: MS2130 (lsusb: Bus 002 Device 005: ID 345f:2130 MACROSILICON USB3.0 UHD) 

Settings:
- Quarter of frame mode: Checked
- HDR to SDR Tone Mapping: Checked
- Resolution/FPS: 1920*1080/60
- video format: yuyv
- Area for LUT: Fullscreen
- Auto signal detection: Checked
- Signal Channel: 240
- Sparse Processing: Checked

**Comment by awawa-dev** [comment]
Currently do not see the issue here. Disable the signal detection first
`2024-09-27T11:22:54.902Z [V4L2:AUTO] The video control requested for the grabber restart due to signal lost`

**Reply by brfield94** [reply]
Automatic signal detection disabled, I will continue to test.

**Reply by brfield94** [reply]
That seems to have fixed it, haven't seen it happen in a few hours.
